auth/auth-model.js

```diff
@@ -0,0 +1,26 @@
+const db = require("../database/dbConfig")
+
+module.exports = {
+  add,
+  findById,
+  findBy,
+};
+
+function findBy (filter) {
```

Comment:
This is short right now, so it doesn't really matter, but on full projects: I would recommend keeping the functions in this file organized. Alphabetically, by behaviour (this is what I do), by something.

By "by behaviour", I mean I would put `find` then `findById` (since its a special case of find), then put `add` (since it doesn't really related to either find), then `delete` (since its related to add), etc.